discovery.md

```diff
@@ -691,108 +695,102 @@ Content-Type: application/json
 ]
 ```
 
-The format of the HTTP Response MUST adhere to the following:
+The format of the `200 OK` HTTP Response MUST adhere to the following:
 ```
+200 OK
 Content-Type: application/json
 
 [
-  "id": "{id}"
+  {
+    id: "{id}",
+    epoch: "{epoch}"
+  },
   ...
 ]
 ```
+#### `DELETE /services`
```

Comment:
Should we support a DELETE w/o a body too - meaning put the id/epoch in the URL ?  For the singleton case it might be easier.